app/requirements/base.txt

```diff
@@ -1,7 +1,30 @@
+alembic==1.8.1
+anyio==3.6.2
+asgiref==3.5.2
+asyncpg==0.26.0
+click==8.1.3
+colorama==0.4.6
+dnspython==2.2.1
+email-validator==1.3.0
 fastapi==0.79.0
-uvicorn[standard]==0.17.6
```

Comment:
Please don't overwrite the requirements with the output of freeze. Only add the dependency and version.